modules/webapp/src/main/elm/Messages/Comp/ItemCard.elm

```diff
@@ -43,9 +43,9 @@ de =
     { basics = Messages.Basics.de
     , dueOn = "Fällig am"
     , new = "Neu"
-    , openAttachmentFile = "Öffne Anhang"
-    , gotoDetail = "Gehe zur Detailansicht"
-    , cycleAttachments = "Gehe durch Anhänge"
+    , openAttachmentFile = "Anhang öffnen"
+    , gotoDetail = "Detailansicht"
+    , cycleAttachments = "Anhänge durchschalten"
```

Comment:
This is hard to translate into German, imho :) tbh I wouldn't know what that means (but the former was not better at all)